### DISCUSSION TITLE
[Request] Steam Controller wireless receiver

### DISCUSSION THREAD

**Discussion by cualquiercosa327** [cathery/sys-con#110] [Controller requests]
Hello,i know they are some request to support the Steam Controller with normal fimware,but this is diferent.
The Steam Controller can be used as a Switch Pro Controller if you install a modded fimware on it working on every Switch Fimware (without need Atmosphere): 

https://github.com/greggersaurus/OpenSteamController

but it is limited to be conected the controller to a free usb port (The author still has no look on BT).
My question is if the usb wireless receiver could be implemented on sys-con to be detected and this way could connect the Steam Controller "modded" wireless.
I am no sure if this would work this way without add some code to the open fimware to contact with the usb wireless.
Thank you

**Comment by noemu** [comment]
If you connect the controller via dongle on a computer, is the controller detected as a steam controller?

**Comment by cualquiercosa327** [comment]
Hello,i have just test it.
On windows if i conect it without conect the Steam Controller ,it seems as if there were no controller.Once i link the Steam Controller with it,it opens Steam and says i have a controller installed.

On linux,it is detected as soon as i connect the usb.I attach the dmesg and lsusb output
[steam.txt](https://github.com/cathery/sys-con/files/4020380/steam.txt)

**Comment by cualquiercosa327** [comment]
The applet output
![1](https://user-images.githubusercontent.com/16308276/71742342-b1ec9a00-2e61-11ea-8991-7fdc932da2da.jpg)
![2](https://user-images.githubusercontent.com/16308276/71742343-b1ec9a00-2e61-11ea-858c-d688c08632a5.jpg)
